src/axom/quest/examples/quest_distributed_distance_query_example.cpp

```diff
@@ -963,29 +906,29 @@ class QueryMeshWrapper
    * - check that points within threshold have a closest point
    *   on the object.
    * - check that found closest-point is near its corresponding
-   *   closest point on the circle (within tolerance)
+   *   closest point on the sphere (within tolerance)
    *
    * Return number of errors found on the local mesh partition.
    * Populate "error_flag" field with the number of errors, for
    * visualization.
    *
-   * Randomized circle points (--random-spacing switch) can cause
+   * Randomized object points (--random-spacing switch) can cause
```

Comment:
Just a style note, but I'd suggest rephrasing to "Randomized points on the sphere" rather than the severely-overloaded word "object".

Reply:
Good suggestion.